model/components/explorviz.yml

```diff
@@ -0,0 +1,18 @@
+# Explorviz
+  - id: explorviz
+    name: ExplorViz
+    description: ExplorViz is a monitoring and visualization approach, which uses dynamic analysis techniques to provide a live trace visualization of large software landscapes. For monitoring purposes, ExplorViz employs the Kieker monitoring agent.
+    link: https://www.explorviz.net
+    github:
+      - ExplorViz
```

Comment:
please provide the path to the repository or repositories (if there are multiple), not the organization!
for example:
- ExplorViz/ExplorViz
- ExplorViz/explorviz-frontend
- ...